### PR TITLE
Inventory Issue Page

### PR DESCRIPTION
**Pull request by TimeBones**
Inventory Issues page added
Inventory checked when opened and right before checkout
Any issues found will open inventory issues page
Options for removing individual line items, fixing all quantities on order, and returning to cart

### CODE REVIEW

File: one_click/client/components/OneClickLayout/components/CheckoutForm/CheckoutForm.jsx [hunk 1]
```diff
@@ -1,11 +1,25 @@
-import React  from 'react';
+import React, { useEffect }  from 'react';
 import { Route, useLocation, Switch } from "react-router-dom";
 import { TransitionGroup, CSSTransition } from "react-transition-group";
-import { BillingAddress } from '@boldcommerce/checkout-react-components';
+import { BillingAddress, useLineItems } from '@boldcommerce/checkout-react-components';
 import { Shipping } from '../Shipping';
+import { useAnalytics, useInventory } from '../../../../hooks';
+import { Inventory } from '../Inventory/Inventory';
```
Comment (CalinR) [inline one_click/client/components/OneClickLayout/components/CheckoutForm/CheckoutForm.jsx:7]: Once you update the suggested change in one_click/client/components/OneClickLayout/components/Inventory/Index.js, you should change this to
```
import { Inventory } from '../Inventory';
```

This is just to keep this standardized with the rest of the project.

**Review by CalinR — CHANGES_REQUESTED** [review]
@TimeBones I've found a few things for you to look at

File: one_click/client/components/OneClickLayout/OneClickLayout.jsx [hunk 2]
```diff
@@ -25,6 +25,10 @@ const OneClickLayout = ({ orderStatus }) => {
         (isProcessing && <OrderProcessing />) || (
           <>
             <div className="Checkout__Main">
+              {
+                Object.keys(orderErrors).some((errorKey) => orderErrors[errorKey] !== null) && 
+               <Message type="alert">An error with your order has occured, please try again</Message>
```
[the remark this answers is not in the document]
Comment (TimeBones) [reply one_click/client/components/OneClickLayout/OneClickLayout.jsx]: Changed from checking for any error to only checking for order errors

File: one_click/client/components/OneClickLayout/components/Inventory/Inventory.jsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+                    orderQty={item.product_data.quantity}
+                    stockQty={item.product_data.stock}
+                    image={item.product_data.image_url}
+                    onRemove={() => removeLineItem(item.product_data.line_item_key) }
+                />);
+        }
+    })
+
+    const handleUpdateQuantities = () => {
+        lineItems.map((item) => {
```
[the remark this answers is not in the document]
Comment (TimeBones) [reply one_click/client/components/OneClickLayout/components/Inventory/Inventory.jsx]: Changed `handleUpdateQuantities` to async function to await for all promises before `history.push`

**Pull request merged**